spec/models/genre_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Genre, type: :model do
```

Comment:
you can omit the `RSpec` here, as well as the `type` argument. 

`describe Genre do` works just as well :)